app/scripts/constants.js

```diff
@@ -84,7 +84,10 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
 
   // This blacklist hides certain kinds from the "Other Resources" page because they are unpersisted, disallowed for most end users, or not supported by openshift but exist in kubernetes
   AVAILABLE_KINDS_BLACKLIST: [],
-
+  // The long running watch on events may be extremely expensive.
+  // This currently by the notificaiton drawer to setup a long running
+  // watch on events
```

Comment:
Sounds good. Tempted to eliminate the comment completely as redundant, other than the `by the drawer` part.